js/src/components/tree-select-control/options.js

```diff
@@ -46,8 +46,11 @@ const Options = ( {
 		);
 	};
 
+	const isRoot = ( option ) => option.value === 'all';
```

Comment:
❓ Similar comment to the previous one: what would happen if consumer provides an option with the value "all"? It wouldn't work as expected when users click on the consumer-provided option with value "all".